script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -569,12 +578,20 @@ But I can't take my eyes off him.
 [sha:18d375d3f454f6fbb5a07daa827269c28e34c804]{
 -- Page 839, Offset 7891.
 -- あの底なしを無視できない。
-I cannot ignore the bottomless void in front of me.
+// Plural since he's referring to Vlov's eyes here
+I cannot ignore the bottomless voids in front of me.
 }
 [sha:7bd2a1ab8c15e5cbee0eb0873c5892265d7f6cb7]{
 -- Page 840, Offset 7892.
 -- アレは違う。白い女の言っていた<魔|も><眼|の>とは、きっと根本が違っている。見る者を金縛りにする力なんてあの眼にはありはしない。なぜならアレは狂躁の眼だ。気が違っているモノの<眼|まなこ>だ。衝動に押し流されるだけの、<醜|みにく>く無様な、自らの異常に振り回される、俺と同じ人でなしの―――
-That can't be right. This must be fundamentally different from the Mystic Eyes that the woman in white talked about. These eyes don't have the power to paralyze a person. Because they are eyes of madness. They're the eyes of a madman. An ugly and pitiful being that flourishes on his own abnormality and is driven by nothing but his impulses. A bastard just like me―――
+// Added newline to avoid awkward text break
+His eyes are different.
+ They must be fundamentally different to the Mystic Eyes that the woman in white was talking about.
+ These eyes don't have the power to paralyze at all.
+ Because these are the eyes of madness.
```

Comment:
no need for 'the', makes it sound as if these eyes are special when he really is just saying they're mad.